tests/Responses/Expected/cart/german_filled_cart_with_simple_product_summary_response.json

```
@@ -40,7 +40,9 @@
                     {
                         "code": "MUG_MATERIAL_CODE",
                         "name": "Becher Material",
-                        "value": "Wood"
+                        "value": [
```

I was thinking about how to solve it. What do you think about something like that:
```suggestion
                        "type": list
                        "value": [
```
and 
```suggestion
                        "type": scalar
                        "value": "Wood"
```

And later it could be extended by custom types, like object or whatever